node/src/neighborhood/gossip_producer.rs

```diff
@@ -335,46 +336,37 @@ mod tests {
     }
 
     #[test]
-    fn produce_debut_or_ipchange_creates_a_gossip_to_a_target_about_ourselves_when_accepting_connections(
+    fn produce_debut_or_ipchange_creates_properly_formed_gossip_with_node_addr_opt_if_we_accept_connections_through_the_firewall(
```

Comment:
I don't like the phrase `...with_node_addr_opt_` because, after all, the true meaning is that the node_addr_opt is Some(...) and on the contrary , `...without_node_addr_opt_` is simply None. 

Partially I feel the problem lies in the word `opt` and should be dropped. `...with_node_addr_present...` is is cleaner to me in overall.

Reply:
Okay...